libs/database/src/lib/schemas/ads-campaign.schema.ts

```diff
@@ -23,7 +23,7 @@
 
 import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
 import { SchemaTypes } from 'mongoose';
-import { Account, CastcleBase } from '.';
+import { User, CastcleBase } from '.';
```

Comment:
```suggestion
import { CastcleBase } from './common.ts';
import { User } from './user.ts';
```

circular
user -> index -> ads